core/templates/pages/skill-editor-page/editor-tab/skill-misconceptions-editor/skill-misconceptions-editor.component.html

```diff
@@ -0,0 +1,108 @@
+<div class="oppia-editor-card-with-avatar oppia-mobile-collapsible-card">
+  <div class="oppia-editor-card-body">
+    <div class="oppia-editor-card-section skill-misconception-content">
+      <div class="oppia-misconception-card-header oppia-mobile-collapsible-card-header" (click)="toggleMisconceptionLists()">
+        <strong class="heading-text">Misconceptions</strong>
+        <i class="fa fa-caret-down"
+           *ngIf="!misconceptionsListIsShown"
+           aria-hidden="true">
```

Comment:
Should this be `[attr.aria-hidden]`? Ditto below.

Reply:
I think both can work as we are assigning a value here not any variable. Still, updated as per suggestion.
Thanks